config/initializers/z_11_plugin_templates.rb

```diff
@@ -7,7 +7,7 @@
 
 
 # Unless the DB is already migrated, do nothing
-if Configuration.table_exists?
+if (ActiveRecord::Base.connection rescue false) && Configuration.table_exists?
```

Comment:
@aapomm What's the purpose of this part added here?

Reply:
> Rails 5.2 now loads the environment on all db rake tasks. Previously it did
> not for setup or create. This means if the db doesn't exist this code will
> fail. Technically we shouldn't communicate with AR during initializers so this
> is workaround.

Note that this is a backport from Pro.